lighthouse-cli/test/fixtures/dobetterweb/dbw_tester.html

```diff
@@ -225,6 +225,21 @@ <h2>Do better web tester page</h2>
 <button class="small-button">Do something</button>
 <button class="small-button">Do something else</button>
 
+<!-- FAIL - frame-focusable-content -->
+<iframe tabindex="-1" src="focusable-iframe.html"></iframe>
```

Comment:
use srcdoc instead to avoid the 1-line extra file? :)

Reply:
That was my first try, didn't work! Not supported I suppose.

Reply:
> That was my first try, didn't work! Not supported I suppose.

interesting, are srcdoc iframes a unique origin and so not available in the parent via js (so axe wouldn't be able to see inside)?